src/sentry/apidocs/hooks.py

```diff
@@ -297,27 +297,26 @@ def _fix_issue_paths(result: Any) -> Any:
     """
     items = list(result["paths"].items())
 
-    deleted_paths = []
     modified_paths = []
 
     for path, endpoint in items:
         if "{var}/{issue_id}" in path:
-            if path.startswith("/api/0/organizations/{organization_id_or_slug}/"):
-                deleted_paths.append(path)
-            else:
-                modified_paths.append(path)
-
-    for path in deleted_paths:
-        del result["paths"][path]
+            modified_paths.append(path)
```

Comment:
Added this hook in https://github.com/getsentry/sentry/pull/78483 to make it possible to document the `/issues|groups/{issue_id}` endpoints. Didn't test it with the real docs though so it needed a small change to work correctly